Neos.Flow/Tests/Functional/Mvc/ActionControllerTest.php

```diff
@@ -286,6 +286,28 @@ public function validatedGroupObjectArgumentsAreValidated()
         $this->assertEquals($expectedResult, $response->getContent());
     }
 
+    /**
+     * @test
+     */
+    public function cyclicCollectionsAreValidated()
```

Comment:
This test will fail and let this object through validation, even though the inner e-mail is invalid, without the push/pop semantic applied.